includes/subscriptions.php

```diff
@@ -280,7 +280,16 @@ function send_notifications( $post_id ) {
 		$request = wp_remote_post(
 			untrailingslashit( $target_url ) . '/wp/v2/dt_subscription/receive',
 			[
-				'timeout' => 5,
+				/**
+				 * Filter the timeout used when calling `\Distributor\Subscriptions\send_notifications`
+				 *
+				 * @hook dt_subscription_post_timeout
+				 *
+				 * @param {int} $timeout The timeout to use for the remote post. Default `5`.
+				 *
+				 * @return {int} The timeout to use for the remote post.
```

Comment:
IMO, we should follow phpDoc standard here by removing `{}` from the types

Reply:
@dinhtungdu removed braces and added tag for $post parameter.